packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -512,12 +512,11 @@ library ChainAssertions {
         }
     }
 
-    /// @dev Asserts that for a given contract the value of a storage slot at an offset is 1.
-    function assertSlotValueIsOne(address _contractAddress, uint256 _slot, uint256 _offset) internal view {
+    /// @dev Asserts that for a given contract the value of a storage slot at an offset is 1 or 0xff.
+    ///      A call to `initialize` will set it to 1 and a call to _disableInitializers will set it to 0xff.
+    function assertSlotValueIsSet(address _contractAddress, uint256 _slot, uint256 _offset) internal view {
```

Comment:
nit: Worth renaming this to `assertInitializedSlotIsSet` so it's super clear this is meant to check the initializer